tools/GenerateDocFxStructure/Program.cs

```diff
@@ -0,0 +1,313 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Web;
+using CommandLine;
+using GenerateDocFxStructure.Helpers;
+
+namespace GenerateDocFxStructure
+{
+    class Program
+    {
+        private static readonly string[] _mediaExtentions = new string[] { ".png", ".jpg", ".jpeg", ".mpg", ".mpeg", ".mov", ".mp4", ".bmp", ".gif", ".fzz" };
```

Comment:
I don't think we should be copying .fzz. Honestly I'd instead show warnings if you see any of:
- .jpeg (you should use .jpg instead)
- .fzz (we should not point to fzz anywhere, it's a source for the diagram picture)